datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/scale/event/DatalakeHorizontalScaleValidationFailedEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.datalake.flow.datalake.scale.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.datalake.flow.SdxContext;
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class DatalakeHorizontalScaleValidationFailedEvent extends SdxEvent {
```

Comment:
You have a lot of events, are they all needed? Or can we reuse some of them for different purposes. I mainly bring this up since it can be a pain to make changes if we have a ton of files to be aware of and take care of.